share/man/man4/ix.4

```diff
@@ -74,25 +74,70 @@ For information on enabling VLANs, see
 .Sh HARDWARE
 The
 .Nm
-driver supports the following cards:
+driver supports Intel 10Gb Ethernet PCIe adapters,
+including:
 .Pp
 .Bl -bullet -compact
 .It
-Intel(R) 10 Gigabit XF SR/AF Dual Port Server Adapter
+Intel(R) 82598EB 10 Gigabit Ethernet (10Gb/1Gb)
 .It
-Intel(R) 10 Gigabit XF SR/LR Server Adapter
-.It
-Intel(R) 82598EB 10 Gigabit AF Network Connection
-.It
-Intel(R) 82598EB 10 Gigabit AT CX4 Network Connection
+Intel(R) Ethernet X520 (82599) (10Gb/1Gb)
+.It 
+Intel(R) Ethernet X520 Bypass (82599) (10Gb/1Gb)
+.It 
+Intel(R) Ethernet X540 (10Gb/1Gb/100Mb)
+.It 
+Intel(R) Ethernet X540 Bypass (10Gb/1Gb/100Mb)
+.It 
+Intel(R) Ethernet X550 (10Gb/5Gb/2.5Gb/1Gb/100Mb)
+.It 
+Intel(R) Ethernet X552 (10Gb/5Gb/2.5Gb/1Gb/100Mb)
+.It 
+Intel(R) Ethernet X553 (10Gb/5Gb/2.5Gb/1Gb/100Mb)
```

Comment:
I would remove all the listed speeds because I think all of these adapters can go all the way down to 10mbit with a copper PHY and should be able to do 100mbit SERDES.

It would be helpful to document the 2.5/5Gb capable adapters including the sysctl dev.ix.0.advertise_speed to enable them in auto-negotiate but maybe in a follow up PR to move this along.